spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/OpenTelemetryConfigurationsMicrometerConfigurationTests.java

```diff
@@ -99,6 +101,12 @@ void shouldBackOffOnCustomBeans() {
 		});
 	}
 
+	@Configuration(proxyBeanMethods = false)
+	@EnableConfigurationProperties(TracingProperties.class)
```

Comment:
This is a bit unusual. Why is a user configuration class enabling one of our configuration properties classes?

Reply:
That's because Moritz has created a test for a subconfiguration only that now requires the TracingProperties and those are enabled in a higher order configuration.